src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/RegionalProcessingCenter.java

```diff
@@ -32,25 +33,28 @@ public RegionalProcessingCenter(@JsonProperty("faxNumber") String faxNumber,
         this.postcode = postcode;
         this.city = city;
         this.email = email;
+        this.hearingRoute = hearingRoute;
     }
 
-    private String faxNumber;
```

Comment:
Why were these removed?

Reply:
The class uses lombok `@value` annotation which makes fields immutable (gives them private access and final keyword) `@value` is shorthand for `final @ToString @EqualsAndHashCode @AllArgsConstructor @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE) @Getter`

Reply:
Sorry yeah I replied on the last PR that it automatically sets it but closed that PR